libs/shared/ui-components/src/Header/Header.tsx

```diff
@@ -21,7 +21,7 @@ export const Header: FC<THeaderProps> = ({
   return (
     <header
       className={clsx(
-        'fixed inset-x-0 top-0 z-20 mx-auto text-white sm:absolute',
+        'fixed inset-x-0 top-0 z-20 mx-auto text-[#efefef] sm:absolute',
```

Comment:
Improve white text on black background in the header (both Labs and Consulting).